error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/Slf4jLoggerDeclaration.java

```diff
@@ -53,7 +53,7 @@ public final class Slf4jLoggerDeclaration extends BugChecker implements Variable
   private static final Matcher<ExpressionTree> IS_GET_LOGGER =
       staticMethod().onDescendantOf("org.slf4j.LoggerFactory").named("getLogger");
   private static final String CANONICAL_STATIC_LOGGER_NAME_FLAG =
-      "Slf4jLogDeclaration:CanonicalStaticLoggerName";
+      "Slf4jLoggerDeclaration:CanonicalStaticLoggerName";
```

Comment:
The `git grep -l Slf4jLogDeclaration | xargs sed -i 's,Slf4jLogDeclaration,Slf4jLoggerDeclaration,'` changes should be moved to a separate PR. I can do that, but CC @mohamedsamehsalah if you're up for it.

Reply:
File it: #1436. Let's make sure to merge that PR first.